test/functional/update/update-type-changes-file-to-symlink.bats

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env bats
```

Comment:
Test looks good, but it's a good idea to add a comment on all file type changes being tested so we can check if there's nothing left.